### ISSUE TITLE
Adjust transition from program cards to 'Leading by Example'

### ISSUE THREAD

**Issue by haleyjohnso** [vigetlabs/npm#335]
## Overview

- **URL:** https://www.nationalpublicmedia.com/
- **Size:** 1
- **Level of urgency:** [Low]
- **Potential deadline:** [6/19]
- **Harvest task:** National Public Media [NPM] > [01] Redesign & Support

**Page or Element Name**
Homepage, program cards block and featured content 'Leading by Example' block.

**Example URL**
https://www.nationalpublicmedia.com/
	
## Details

**Summary:**

The two blocks having the same background color makes it appear as if they are related and we'd like there to be some sort of separation. 

**Current vs. Desired state:**
Currently, the two blocks have the same background, and we'd like to have some sort of separation. 

On the podcast sponsorship page (https://www.nationalpublicmedia.com/sponsorship/podcast-sponsorship/) the program block appears to have a rule underneath it that separates it from the featured work block below (see image.png attached).

We would like to make this part of the program card block on the homepage also so that there is separation between the program card block and the featured content / 'Leading by Example' block.

**Screenshots**
Attached.
![image (5)](https://user-images.githubusercontent.com/57419582/84231502-88d4d300-aaab-11ea-841d-5560826824a0.png)
![Screen_Shot_2019-11-14_at_2 38 43_PM](https://user-images.githubusercontent.com/57419582/84231525-97bb8580-aaab-11ea-9119-ae7aa36f9fb1.png)

**Comment by haleyjohnso** [comment]
Hey @ten1seven they want to keep the block's background gray on the homepage for consistency with other pages and so the cards stand out. They're wondering if we add a thin rule under the programs block so that it can be used next to another block with a gray background

**Comment by ten1seven** [comment]
Yep, that was the original plan. I just wanted to make sure they were aware they had other options.
